modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_aggregator/configuration.jsp

```diff
@@ -61,98 +66,22 @@ if (organizationId > 0) {
 					<aui:button name="removeOrganizationButton" onClick="<%= taglibRemoveFolder %>" value="remove" />
 				</div>
 
-				<aui:script sandbox="<%= true %>">
-					var <portlet:namespace />selectOrganizationButton = document.getElementById(
-						'<portlet:namespace />selectOrganizationButton'
-					);
-
-					<portlet:namespace />selectOrganizationButton.addEventListener(
-						'click',
-						function (event) {
-							Liferay.Util.openSelectionModal({
-								onSelect: function (event) {
-									var form = document.getElementById('<portlet:namespace />fm');
-
-									if (form) {
-										var organizationId = form.querySelector(
-											'#<portlet:namespace />organizationId'
-										);
-
-										if (organizationId) {
-											organizationId.setAttribute('value', event.entityid);
-										}
-
-										var organizationName = form.querySelector(
-											'#<portlet:namespace />organizationName'
-										);
-
-										if (organizationName) {
-											organizationName.setAttribute(
-												'value',
-												event.entityname
-											);
-										}
-									}
-
-									Liferay.Util.toggleDisabled(
-										'#<portlet:namespace />removeOrganizationButton',
-										false
-									);
-								},
-
-								<%
-								String portletId = PortletProviderUtil.getPortletId(User.class.getName(), PortletProvider.Action.VIEW);
-								%>
-
-								selectEventName:
-									'<%= PortalUtil.getPortletNamespace(portletId) %>selectOrganization',
-
-								title:
-									'<liferay-ui:message arguments="organization" key="select-x" />',
-
-								<%
-								PortletURL selectOrganizationURL = PortletProviderUtil.getPortletURL(request, Organization.class.getName(), PortletProvider.Action.BROWSE);
-
-								selectOrganizationURL.setParameter("tabs1", "organizations");
-								selectOrganizationURL.setWindowState(LiferayWindowState.POP_UP);
-								%>
-
-								url: '<%= selectOrganizationURL.toString() %>',
-							});
-						}
-					);
-				</aui:script>
-
-				<aui:script require="metal-dom/src/dom">
-					var dom = metalDomSrcDom.default;
-
-					var <portlet:namespace />selectionMethod = document.getElementById(
-						'<portlet:namespace />selectionMethod'
-					);
-
-					if (<portlet:namespace />selectionMethod) {
-						<portlet:namespace />selectionMethod.addEventListener('change', function (
-							event
-						) {
-							var usersSelectionOptions = document.getElementById(
-								'<portlet:namespace />usersSelectionOptions'
-							);
-
-							if (usersSelectionOptions) {
-								var showUsersSelectionOptions = !(
-									<portlet:namespace />selectionMethod.val() === 'users'
-								);
-
-								if (showUsersSelectionOptions) {
-									dom.addClasses(usersSelectionOptions, 'hide');
-								}
-								else {
-									dom.removeClasses(usersSelectionOptions, 'hide');
-								}
-							}
-						});
-					}
-				</aui:script>
+				<liferay-frontend:component
```

Comment:
I think a better approach for this can be to replace `aui:button` with `clay:button` and use the `propsTransformer` method (or some additional abstraction layer to make it simpler) to set the `click` listener.

Reply:
Makes sense! I didn't saw the complete context, I just forget the aui:button on top of that :P